src/index.js

```diff
@@ -1,78 +1,125 @@
 const compiler = require('vue-template-compiler');
 const transpile = require('vue-template-es2015-compiler');
-const {spawnSync} = require('child_process');
-const {join, dirname} = require('path');
-const {readFileSync} = require('fs');
+const { spawnSync } = require('child_process');
+const { join, dirname } = require('path');
+const { readFileSync } = require('fs');
+const postcss = require('postcss')
+const postcssModules = require('postcss-modules-sync').default
 const exportsTarget = '((module.exports.default || module.exports).options || module.exports.default || module.exports)';
 const defaultConfig = {
-  transpileTemplates : true
+  transpileTemplates: true
 };
 let globalConfig = defaultConfig;
 
 module.exports = function ({ content, filename, hook }) {
+
   function retrieveContent(config) {
-    if (config.content){
+    if (config.content) {
       return config.content;
-    }else if (config.attrs.src){
+    } else if (config.attrs.src) {
       let fullPath = join(dirname(filename), config.attrs.src);
       return readFileSync(fullPath, 'utf8');
-    }else{
+    } else {
       return '';
     }
   }
 
-  const {template, script} = compiler.parseComponent(content, {pad : 'line'});
+  function retrieveAndTranspileContent(config, noTranspileLangs, transpileSpecial) {
```

Comment:
the `noTranspileLangs` isn't being used, presumably it should be doing something like this old line:
```js
iff (script.attrs.lang && !['js', 'javascript'].includes(script.attrs.lang.toLowerCase())){
```

Reply:
Right, will fix in a moment.

Reply:
Done.